### ISSUE TITLE
[ENHANCEMENT] Visible real-time log when doing operations on packages

### ISSUE THREAD

**Issue by redactedscribe** [marticliment/UniGetUI#1711]
### Please confirm these before moving forward

- [X] I have searched for my feature proposal and not found a work-in-progress/duplicate/resolved/discarded issue.
- [X] This improvements refers to an existing feature. If you want to suggest a new feature, please use [this template](https://github.com/marticliment/WingetUI/issues/new?assignees=marticliment&labels=new-feature&projects=&template=feature-request.yml&title=%5BFEATURE+REQUEST%5D+%28Enter+your+description+here%29)
- [X] This improvements is not a bug. If you want to report a bug, please use [this template](https://github.com/marticliment/WingetUI/issues/new?assignees=marticliment&labels=bug&projects=&template=bug-issue.yml&title=%5BBUG%5D+%28Enter+your+description+here%29)

### Describe the improvement

Some months ago an Operation history (log) feature was introduced to the UI (#1023), added as a menu item to the top-right. This is a useful addition however it is both tucked away and also not real-time. For its purposes, this is fine, but I think it would be nice to have a glimpse into this log as packages are being installed/uninstalled etc.

I imagine a "Show log" / "Show operations" button at the top (same row as "Help") which we could click to reveal an integrated, small by default, but draggable (vertically expandable) log window which updates in real time, and would appear at the very bottom of WingetUI's UI, below any info rows:

![wingetui_2024-01-17_16-14-28](https://github.com/marticliment/WingetUI/assets/25013746/f94609dd-0289-456a-b912-e085644b7a3b)

Additionally, each row (see screenshot) could have its own log button, say an arrow/log/console icon, which when clicked, either expands a similar integrated log, or pops up a window, displaying only log lines related to the operation. A window is the cleaner suggestion, but if it's possible, a third suggestion would be for this button to instead jump to the relevant point in the real-time log already on display (as suggested in the previous paragraph). In that case, a jump-to-top button on the real-time log would be useful to return to the current line of output.

In short, I'd like to see what's happening under-the-hood when packages are being operated upon. Keeping the visibility of this log optional means those who prefer the less noisy and simplified experience can keep it that way.

### Describe how this improvement could help users.

As packages are installed/uninstalled etc, we could see the details of the paths being affected, potentially cancel the operation if we notice it's not what we thought we were changing, etc. After some years of dealing with Scoop via the CLI, it's proven to be useful on many an occasion to see these details.

**Comment by redactedscribe** [comment]
Wasn't looking for it (I did try) but it looks like issue #329 already existed. I'm not sure where this "live output/log" button is, but it was apparently added in https://github.com/marticliment/WingetUI/commit/7955889e216fb3cdf73b82754391c30f353a00c4. Was it removed, or where can it be found?

**Comment by marticliment** [comment]
> Wasn't looking for it (I did try) but it looks like issue #329 already existed. I'm not sure where this "live output/log" button is, but it was apparently added in [7955889](https://github.com/marticliment/WingetUI/commit/7955889e216fb3cdf73b82754391c30f353a00c4). Was it removed, or where can it be found?

Click on the progressbar shown on an installation widget.

**Comment by redactedscribe** [comment]
> Click on the progressbar shown on an installation widget.

Thank you, I see that it creates a pop-up window. Useful, although one must click on each widget to keep a track of what's happening for each package, or switch to the Operation history tab (but as stated, this feels tucked away to me). If you don't consider the other suggestions outlined in this issue worth pursuing, specifically an integrated operation history log for the install/uninstall UIs, you can close it.

**Comment by marticliment** [comment]
I am afraid that due to the parallel-operation direction that UniGetUI is taking an unified log window wouldn't be a good idea, as it would be confusing. 

Details about the new features for package operations can be checked here: https://github.com/marticliment/UniGetUI/pull/3070
